README.md

```diff
@@ -1,128 +1,207 @@
+# **thprac**
+> thprac is a tool for practicing Touhou shmups.
 
-Or: Run the game in whatever way you want (be it through thcrap, with vpatch, through Steam, etc...), then run the thprac exe. Thprac will automatically detect the running game and attach itself to it. You may need to return to the main menu for thprac to take effect.
+[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7O03J4)
 
-Or: Use the launcher. This video explains how the launcher works.
+## **Table of Contents**
+* [Downloading](#downloading)
+* [Usage](#usage)
+* [Compatability](#compatability)
+* [Features](#features)
+* [Backspace Menu](#backspace-menu)
+* [Advanced Options](#advanced-options)
+* [Statistics](#statistics)
+* [Game Specific Features](#game-specific-features)
+* [Credits](#credits)
+* [FAQ](#faq)
+
+## <a name="downloading"></a> **Downloading**
+If you want to download the tool, you can download the [latest release](https://github.com/touhouworldcup/thprac/releases/latest) here and the [latest beta](https://nightly.link/touhouworldcup/thprac/workflows/main/master/thprac.zip) here.
+
+## <a name="usage"></a> **Usage**
+You can use this tool in a variety of ways; the main ways to do this are **placing the executable in the same folder as your game**, **running the executable after running your game**, or **using the thprac launcher**.
+
+### **Using the folder**
+**This method will NOT work with Steam games.** You can run the thprac tool by placing the `thprac.exe` file in the same folder as your game. This method will also automatically detect vpatch. You must ensure that the game is updated to the latest version and follows one of the following naming schemes. This method also works with vpatch.
+- thXX.exe (most games)
+- 東方紅魔郷.exe (Embodiment of Scarlet Devil)
+- alcostg.exe (Uwabami Breakers)
+
+### **Running the Executable**
+**This method will work for most methods.** First, run the game in whichever method you want (whether it's thcrap, using vpatch, Steam, etc.) and then run `thcrap.exe`. thcrap will then detect the running game and attempt to attach itself to it. You may have to return to the menu screen to see thprac take effect.
+
+### **Using the Launcher**
+This video explains how to use the launcher.
 [![A quick and dirty overview of thprac 2.0](https://i.ytimg.com/vi/sRV4PDQceYo/maxresdefault.jpg)](https://www.youtube.com/watch?v=sRV4PDQceYo)
 
-## Compatibility
-thprac officially supports every version of Windows starting with **Windows Vista**. Windows XP may work using [One-Core-API](https://github.com/Skulltrail192/One-Core-API-Binaries), but this is NOT being actively tested.
+## <a name="compatability"></a> **Compatability**
+thprac is compatible with all Windows versions proceeding and including Windows Vista. You may attempt to apply thprac on Windows XP using the [One-Core-API](https://github.com/Skulltrail192/One-Core-API-Binaries) but this is not guaranteed to work.
```

Comment:
I like what I previously wrote here better. For reference, that would be "thprac officially supports every version of Windows starting with Windows Vista. Windows XP may work using [One-Core-API](https://github.com/Skulltrail192/One-Core-API-Binaries), but this is NOT being actively tested."